src/main/scala/com/socrata/geospace/lib/regioncache/HashMapRegionCache.scala

```diff
@@ -54,4 +55,13 @@ class HashMapRegionCache(config: Config) extends MemoryManagingRegionCache[Map[S
       .collect { case (key: RegionCacheKey, Some(Success(index))) => (key, index.size) }
       .sortBy(_._2)
       .reverse
+
+  /**
+   * Returns indices in descending order of Age, not in constant time
+   * @return Indices in descending order of Age
+   */
+  override def keysByLeastRecentlyUsed(): Seq[RegionCacheKey] = {
+    cache.ascendingKeys().toSeq.asInstanceOf[Seq[RegionCacheKey]]
```

Comment:
We could do that, i suppose, though the route asks for a list of items, not necessarily in the order they will be eliminated. But to preserve the intention here Ill make the appropriate change.